website/www/site/content/en/blog/splittable-do-fn-is-available.md

```diff
@@ -0,0 +1,91 @@
+---
+title:  "Splittable DoFn in Apache Beam is Ready to Use"
+date:   2020-12-16 00:00:01 -0800
+categories:
+  - blog
+aliases:
+  - /blog/2020/12/16/splittable-do-fn-is-available.html
+authors:
+  - boyuanzz
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+In 2017, [Splittable DoFn Blog Post](https://beam.apache.org/blog/splittable-do-fn/) proposed
+to build [Splittable DoFn](https://s.apache.org/splittable-do-fn) APIs as the new recommended way of
+building I/O connectors. Splittable DoFn is a generalization of `DoFn` that gives it the core
+capabilities of `Source` while retaining `DoFn`'s syntax, flexibility, modularity, and ease of
+coding. Thus, it becomes much easier to develop complex I/O connectors with simpler and reusable
+code.
+
+Now we are pleased to announce that Splittable DoFn is ready for use in Beam Python/Java/Go SDKs
+starting in version 2.25.0.
+
+Splittable DoFn has three advantages over the existing `UnboundedSource` and `BoundedSource`:
+* Splittable DoFn provides a unified set of APIs to handle both unbounded and bounded cases.
+* Splittable DoFn enables reading from source descriptors dynamically.
+  - Taking Kafka IO as an example, within `UnboundedSource`/`BoundedSource` API, you must specify
+  the topic and partition you want to read from during pipeline construction time. There is no way
+  for `UnboundedSource`/`BoundedSource` to accept topics and partitions as inputs during execution
+  time. But it's native to Splittable DoFn.
```

Comment:
Replace "native" with "built-in" (if that what you mean)

https://developers.google.com/style/word-list#letter-n